packages/contracts-bedrock/test/kontrol/kontrol/run-kontrol.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+export FOUNDRY_PROFILE=kontrol
+
+# Create a log file to store standard out and standard error
+LOG_FILE="run-kontrol-$(date +'%Y-%m-%d-%H-%M-%S').log"
```

Comment:
I think it might be cleaner to put the log file in the `kout` or `kout-logs` directory to keep the root of this package a bit cleaner (if we do `kout-logs` we should make sure that's also in the `.gitignore`)

@tynes let me know if you have any preference here

Reply:
I went ahead and added the logging folder under `tests/kontrol/kontrol/logs`. The reason being we might not want `forge clean` to delete the logs created. Let me know if this is alright with you

Reply:
Good call, I like that